sbt-dotty/sbt-test/sbt-dotty/compiler-plugin/test

```diff
@@ -0,0 +1,4 @@
+> plugin/publishLocal
```

Comment:
Do we have tests that actually run the compiler with a compiler plugin enabled? It'd be good to have something like that in this example project and run them with `> test` here.

Reply:
This project depends on the plugin, thus it's used in the compilation. The plugin will rewrite divide-zero in source code, so that there will be no run-time errors.

Previously, this test (like all sbt scripted tests) only run on nightly. But now I've added this particular one to normal CI.